django/db/models/base.py

```diff
@@ -519,11 +519,14 @@ def __hash__(self):
         return hash(self.pk)
 
     def __reduce__(self):
-        data = self.__dict__
+        data = self.__getstate__()
         data[DJANGO_VERSION_PICKLE_KEY] = get_version()
         class_id = self._meta.app_label, self._meta.object_name
         return model_unpickle, (class_id,), data
 
+    def __getstate__(self):
+        return self.__dict__.copy()
```

Comment:
I'm not sure we need to do this as the default behaviour. A subclass can do it's own `.copy()` (or whatever other method you want to do to build the state)